python/lsst/validate/drp/validate.py

```diff
@@ -334,6 +337,32 @@ def add_measurement(measurement):
     pa1 = measurePA1(metrics['validate_drp.PA1'], matchedDataset, filterName)
     add_measurement(pa1)
 
+    snr_base = [5, 20]
+    name_flux_all = ["base_PsfFlux"]
+    if not skipGalaxies:
+        name_flux_all.append('slot_ModelFlux')
+    for name_flux in name_flux_all:
+        key_model_mag = matchedDataset._matchedCatalog.schema.find(f"{name_flux}_mag").key
+        bin_base = 1
```

Comment:
Insert of paragraph explaining about the binning.  This is pretty different from the way other metrics are calculated so deserves a bit of a special call out.